File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex13.java
//Importando Scanner
import java.util.Scanner;

public class Ex13 {
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	System.out.println("Novo preço de um produto e classificação\n\n");
	
	//vars
	float preco;
	
	//Entrada
	System.out.print("Insira o preço atual do produto: R$ ");
	preco = ent.nextFloat();
	
	//Processamento
	if( preco <= 50 )
		preco *= 1.05;
	else if ( preco <= 100 )
		preco *= 1.1;
	else
		preco *= 1.15;
	
	//Saída
	System.out.println("Novo preço: R$ " + preco);
	
	if ( preco <= 80 )
		System.out.println("Classificação: Barato");
	else if ( preco <= 120 )
		System.out.println("Classificação: Normal");
	else if ( preco <= 200 )
		System.out.println("Classificação: Caro");
	else
		System.out.println("Classificação: Muito Caro");
		
	}

}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex13.c
#include <stdio.h>

main()
{
    printf("\n\n");

    //vars
    float precoI, precoF;


    //Entrada
    printf("Insira o preco inicial do produto: ");
    scanf("%f",&precoI);

    //Processamento
    if ( precoI <= 50 )
        precoF = precoI * 1.05;
    else if ( precoI <= 100 )
        precoF = precoI * 1.1;
    else
        precoF = precoI * 1.15;

    //Saída
    printf("Preco com aumento: R$%.2f\n", precoF);

    if ( precoF <= 80 )
        printf("Classificacao: Barato\n");
    else if ( precoF <= 120 )
        printf("Classificacao: Normal\n");
    else if ( precoF <= 200 )
        printf("Classificacao: Caro\n");
    else
        printf("Classificacao: Muito Caro\n");

}
